src/AppBundle/Entity/Event.php

```
@@ -250,4 +263,29 @@ public function isFull(): bool
 
         return $this->participantsCount >= $this->capacity;
     }
+
+    public function getStatus(): string
+    {
+        return $this->status;
+    }
+
+    public function setStatus(string $status)
+    {
+        $this->status = $status;
```

Can you throw an exception if the status is not valid?